lib/api-syncRecords.js

```diff
@@ -171,15 +171,15 @@ function computeDelta(datasetId, clientRecords, serverRecords) {
  * @param {Object} params the request body, it normally contain those fields:
  * @param {Object} params.query_params the query parameter for the dataset from the client
  * @param {Object} params.meta_data the meta data for the dataset from the client
- * @param {Object} params._fh an object added by the client sdk. it should have the `cuid` of the client
+ * @param {Object} params.__fh an object added by the client sdk. it should have the `cuid` of the client
  * @param {Object} params.clientRecs the client records
  * @param {Function} cb
  */
 function syncRecords(datasetId, params, cb) {
   debug('[%s] process syncRecords request', datasetId);
   var queryParams = params.query_params || {};
   var metaData = params.meta_data || {}; //NOTE: the client doesn't send in this value for syncRecords ATM
-  var cuid = syncUtil.getCuid(params);
+  var cuid = syncUtil.getCuid(params, syncConfig && syncConfig.cuidGenerator);
```

Comment:
Do we need the check for syncConfig existing here? If we do should we also add it here https://github.com/feedhenry/fh-sync/pull/31/files#diff-7be39900ae2242e1d7d427dc7e284b53R64 ?